### PR TITLE
feat: add default for OIDC settings

### PR DESCRIPTION
**Pull request by czosel**
This should help reducing the amount of redundant configuration related
to OIDC.

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -327,6 +330,16 @@ ember_docroot: "{{ ember_basedir }}/htdocs"
 # Set this to the user which apache runs as
 ember_apache_user: www-data
 
+# Keycloak URL
+#TODO trailing slash really needed?
+ember_oidc_host: "{{ camac_auth_idp_url }}/"
```
Comment (czosel) [inline defaults/main.yml:334]: @anehx do you know if the trailing slash is needed?

**Comment by czosel** [comment]
I confirmed that this works on our test system.

File: defaults/main.yml (hunk 1 above)
Comment (anehx) [reply defaults/main.yml:334]: No idea..

Comment (czosel) [reply defaults/main.yml:334]: Let's leave it as is then. I removed the TODO and added a note highlighting that `camac_auth_idp_url` should not have a trailing slash.

**Review by anehx — APPROVED** [review]

**Pull request merged**